Sim/SimpleFtl/SimpleFtl.cpp

```diff
@@ -93,13 +108,16 @@ void SimpleFtl::OnNewCustomProtocolCommand(CustomProtocolCommand *command)
     {
     case CustomProtocolCommand::Code::Write:
     {
-        // Write command to nand
         _RemainingSectorCount = command->Descriptor.SimpleFtlPayload.SectorCount;
         _CurrentLba = command->Descriptor.SimpleFtlPayload.Lba;
-        _ProcessedSectorCount = 0;
-        _PendingCommandCount = 0;
+        _ProcessedSectorOffset = 0;
+        _PendingTransferCommandCount = 0;
+        _PendingNandCommandCount = 0;
+        _ProcessingBlockCount = 0;
+        _BlockBufferCount = 0;
 
-        WriteNextLbas();
+        // Write command to nand
+        HandleWriteCommand();

```

Comment:
We have some method with the "OnEvent" convention and now have "HandleEvent" convention.